pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -83,44 +83,72 @@ func parseGetMetricDataTimeSeries(metricDataResults map[string]*cloudwatch.Metri
 			}
 		}
 
-		series := tsdb.TimeSeries{
-			Tags:   make(map[string]string),
-			Points: make([]tsdb.TimePoint, 0),
-		}
+		// In case a multi-valued dimension is used and the cloudwatch query yields no values, create one empty time series for each dimension value.
+		// Use that dimension value to expand the alias field
+		if len(metricDataResult.Values) == 0 && query.isMultiValuedDimensionExpression() {
+			series := 0
+			multiValuedDimension := ""
+			for key, values := range query.Dimensions {
+				if len(values) > series {
+					series = len(values)
+					multiValuedDimension = key
+				}
+			}
 
-		keys := make([]string, 0)
-		for k := range query.Dimensions {
-			keys = append(keys, k)
-		}
-		sort.Strings(keys)
+			for _, value := range query.Dimensions[multiValuedDimension] {
+				emptySeries := tsdb.TimeSeries{
+					Tags:   map[string]string{multiValuedDimension: value},
+					Points: make([]tsdb.TimePoint, 0),
+				}
+				for key, value := range query.Dimensions {
```

Comment:
I think it would be more explanatory to name it `values`, since it's an array and that's the naming used otherwise in the code:
```suggestion
				for key, values := range query.Dimensions {
```